src/plugins/ui_stateful/clipboard.ts

```diff
@@ -157,9 +156,10 @@ export class ClipboardPlugin extends UIPlugin {
       }
       case "PASTE_FROM_OS_CLIPBOARD":
         this.state = new ClipboardOsState(cmd.text, this.getters, this.dispatch, this.selection);
-        this.state.paste(cmd.target);
+        this.state.paste(cmd.target, { pasteOption: cmd.pasteOption });
         this.lastPasteState = this.state;
         this.status = "invisible";
+        this.allowPaste = true;
```

Comment:
Is that useful ?

Reply:
If we cut something, 'this.allowPaste' will be set to false. When we paste external text now, the first paste works perfectly. When you try to paste that text again, "PASTE" is called, and 'this.allowPaste = false' prevents the content from being pasted. As a result, we set 'this.allowPaste = true' to allow repeated pastes when using external content.